### ISSUE TITLE
Support links to Wiki page & source in another project

### ISSUE THREAD

**Issue by dahovey** [MicrosoftDocs/vsts-docs#7386]
There is some documentation for absolute links that can point to something in another project, but as it relates to the Wiki markdown functionality it appears this is not supported?

I have several projects that use functionality from one shared project. It would be helpful to have links with auto-completion that can point to another Wiki, of another project.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fa622358-f5bb-1f5c-177a-081a37b6acef
* Version Independent ID: 72f7fd16-a822-66c8-3a34-8905645ce5f2
* Content: [Markdown syntax for files, widgets, and wikis - Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/project/wiki/markdown-guidance?view=azure-devops)
* Content Source: [docs/project/wiki/markdown-guidance.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/project/wiki/markdown-guidance.md)
* Product: **devops**
* Technology: **devops-collab**
* GitHub Login: @chcomley
* Microsoft Alias: **chcomley**

**Comment by WilliamAntonRohm** [comment]
@dahovey -- David, thank you for your question.  You may find answers here:
- [Azure DevOps Support Bot](https://azuredevopsvirtualagent.azurewebsites.net/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops) 

Also, you are welcome to [submit your idea for auto-complete links]( https://developercommunity.visualstudio.com/content/idea/post.html?space=21) here. 

@chcomley -- Chrystal, please look into this potential documentation issue.

**Comment by chcomley** [comment]
Hi @dahovey, I've followed up with the PMs and will let you know what I learn. Have a great day.

**Comment by chcomley** [comment]
Hi @dahovey,
It's not clear what kind of link you're trying to embed in the wiki - is it the link for another wiki page or work item? We already support showing the suggestion for wiki pages to link (when you enter **[](/)**,  suggestions come with **/**), but this is scoped within the project only. We don’t support it across the project.

I encourage you to create a suggestion ticket for and upvote it so that we can prioritize. Please go [here](https://developercommunity.visualstudio.com/spaces/21/index.html), and then select **Suggest a feature*.* Thank you! Feel free to @mention me if you have any further questions. Have a great day!

**Comment by dahovey** [comment]
I apologize my question wasn't clear. Yes I frequently use `[go here](/...` auto-completion which works well, but would like this to provide auto-completion to another project, perhaps using another syntax (i.e. `[go here]($/SharedProject/...` where `SharedProject` is another project within organization.

I did find a similar suggestion on the developercommunity site, but it was closed in leu of using full absolute url.

Thanks for your help.
